src/resources/views/crud/columns/date.blade.php

```diff
@@ -5,10 +5,12 @@
 
 <span data-order="{{ $value }}">
     @if (!empty($value))
-	{{
-		\Carbon\Carbon::parse($value)
-		->locale(App::getLocale())
-		->isoFormat($column['format'] ?? config('backpack.base.default_date_format'))
-	}}
+    @php($text = \Carbon\Carbon::parse($value)
```

Comment:
Since we're already modifying all columns, I think it's a good idea to clean them up too. One thing I see in columns that's a little messy is that we didn't follow a clear structure, sometimes we begin with calculating stuff, other times we calculate stuff when echoing - a bit messy.

I think it's a good opportunity to impose this file structure for columns from now on:

```php
{{-- column name --}}
@php
  // TODO: calculate all values here
  $value = 'smth';
  $text = 'whatev';
  $aux = 'whatev';
@endphp

<span>
  // TODO: only echo stuff using $value, $text, $whatev;
</span>
```

And while we're at it, since columns now become a little more complicated, with anchor and everything, I think it would be easier to follow if we just use the ```$column``` array to store the calculated value of the column. So:

```php
{{-- column name --}}
@php
  $column['value'] = \Carbon\Carbon::parse($column['function']($entry));
  $column['text'] = $column['value']->locale(App::getLocale())->isoFormat($column['format'] ?? config('backpack.base.default_date_format')));
@endphp

<span data-order="{{ $column['value'] }}">
  // TODO: echo stuff using $column['value'] or $column['text']
  // TODO: in blade files we include, now we can just pass $column (or better: pass NOTHING, since they will have $column anyway)
</span>
```

What do you think about this @pxpm ? Good idea or bad?